### PR TITLE
Understanding static and dynamic routing

### LINKED ISSUE #1091: Understanding Static and Dynamic routing.
### Brief Summary: 
Every router can learn about remote  network routes in one of two ways:
- Manually or
- Dynamically.
 For Manual networks, the routes are configured by the network administrator, while for dynamic routes, the routes are entered by a dynamic routing protocol

### Key Takeaways: 
By the end of this article, the reader should have the basic knowledge of:
-Static routing
- Types of Static routes
- Dynamic routing
- Types of dynamic routing protocols.
- when to Implement Static routing or Dynamic routing .

### References: 

https://en.wikipedia.org/wiki/Static_routing
https://www.cisco.com/c/en/us/td/docs/switches/datacenter/nexus3000/sw/unicast/503_u1_2/nexus3000_unicast_config_gd_503_u1_2/l3_route.htm

### PR DESCRIPTION
**Pull request by raboba2re**
### Article Checklist
**NOTE: (1st time contributors - submit ONLY one article to get familiar with the process, quality, and style-once published, you can then submit any number of articles.)**
#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.
      
#### Grammar Checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical Checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution Guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**

This fixes #1091

### CODE REVIEW

**Comment by jumamiller** [comment]
@hectorkambow, I'll be reviewing this article!

**Comment by jumamiller** [comment]
@raboba2re, where is the index.md file?

**Comment by raboba2re** [comment]
@jumamiller, It has been updated, sorry for that error.

File: articles/understanding-static-dynamic-routing/index.md [hunk 1]
```diff
@@ -0,0 +1,276 @@
+# Understanding Dynamic and Static Routing
+
+![Hero image](/engineering-education/understanding-dynamic-and-static-routing/hero.jpeg)
+
+[Source](https://unsplash.com/photos/IL1qSqEMNBo)
+
```
Comment (jumamiller) [inline articles/understanding-static-dynamic-routing/index.md:6]: @raboba2re, remove the 'hero' image., should not be included in this file.(Note: DO NOT delete it from your article folder).

File: articles/understanding-static-dynamic-routing/index.md [hunk 2]
```diff
@@ -0,0 +1,276 @@
...
+## Introduction
+
+Different devices work together to ensure that data is transferred from one end device to another, but the router is the primary device that learns which paths are available and which path is best to forward traffic. The mechanism through which the router makes such a decision is known as routing. The path or route through which a packet is sent can either be a static or dynamic route.
+In this article, we will look at static and dynamic routing in details and it is expected that at the end of this article, the reader should know:
```
Comment (jumamiller) [inline articles/understanding-static-dynamic-routing/index.md:10]: Grammar mistakes here, at the most basic level, the active voice emphasizes the person or agent who performs an action, in short, the “actor.” The passive voice emphasizes the recipient of the action or sometimes the action itself.
In this, you've used a lot of passive voice which is not recommended in articles.
Kindly make changes where applicable to ensure the article is in the active voice. 
- Feel free to keep in touch with me on Slack inbox.

File: articles/understanding-static-dynamic-routing/index.md [hunk 3]
```diff
@@ -0,0 +1,276 @@
...
+
+- What static routing is,
+- When to use static routing when setting up a network.
+- The advantages and limitations of static routing
+- What dynamic routing is
+- The advantages and limitations of dynamic routing
+- When to implement static or dynamic routing
+- A brief introduction to some dynamic routing protocols in use today
```
Comment (jumamiller) [inline articles/understanding-static-dynamic-routing/index.md:18]: Punctuations...passive voice mistakes...

File: articles/understanding-static-dynamic-routing/index.md [hunk 4]
```diff
@@ -0,0 +1,276 @@
...
+-Limitations of static routing
+-Dynamic routing protocol
+-Differences between dynamic and static routing
+-Introduction to some dynamic routing protocols (RIP, OSPF, and EIGRP)
+-Limitations of dynamic routing
+-Conclusion
+
+### Functions of a router
+
+Multiple networks are connected using a router. To make a routing decision, the router uses the information in the internet protocol header to make various decisions which include:
```
Comment (jumamiller) [inline articles/understanding-static-dynamic-routing/index.md:38]: This sentence is very long making it hard to read.

**Review by jumamiller — CHANGES_REQUESTED** [review]
Generally, you have a nice article, just fix the long sentences and grammar then we're good to go. Ping me on Slack whenever you're stuck.

File: articles/understanding-static-dynamic-routing/index.md (hunk 2 above)
Comment (raboba2re) [reply articles/understanding-static-dynamic-routing/index.md:6]: @jumamiller , I could not find you on Slack. please what is your username on Slack?

Comment (jumamiller) [reply articles/understanding-static-dynamic-routing/index.md:6]: Hi, have you joined the **#eng-ed** group on slack?

Comment (raboba2re) [reply articles/understanding-static-dynamic-routing/index.md:6]: Yes I have @jumamiller

Comment (raboba2re) [reply articles/understanding-static-dynamic-routing/index.md:6]: With @Rabo James Bature

Comment (jumamiller) [reply articles/understanding-static-dynamic-routing/index.md:6]: What's your username? I couldn't find you either

Comment (jumamiller) [reply articles/understanding-static-dynamic-routing/index.md:6]: Join [here](https://join.slack.com/t/sectionio-community/shared_invite/zt-6174cklt-YFpHmQIAUmc927jqN2EIZQ)

Comment (raboba2re) [reply articles/understanding-static-dynamic-routing/index.md:6]: @Rabo_James_Bature

**Comment by raboba2re** [comment]
@jumamiller I have made the changes.

**Comment by jumamiller** [comment]
@raboba2re, have a look at the image sources, they aren't working!

**Comment by raboba2re** [comment]
> 
> 
> @raboba2re, have a look at the image sources, they aren't working! Better still, reference them at the end of the article!

@jumamiller The author owns the images and I have made changes to reflect that.

File: articles/understanding-static-dynamic-routing/index.md [hunk 5]
```diff
@@ -59,7 +59,7 @@ In the image above, a device in sub-network-A want to reach another in sub-netwo
   For a protocol that uses bandwidth capacity as its metric, the protocol determines the bandwidth capacity of all possible routes to the destination subnet, and the route with higher bandwidth capacity is considered the best path and added to the routing table.
 Because metrics can depend on single or many characteristics, Enhanced Interior Gateway Routing Protocol (EIGRP) uses a composite of bandwidth and delay to calculate its metric.
 ![Bandwidth](/engineering-education/understandinding-dynamic-and-static-routing/bandwidth.jpeg)
-[Image source](AUthor)
+Image source: AUthor
```
Comment (jumamiller) [inline articles/understanding-static-dynamic-routing/index.md]: @raboba2re, "Image source: Author"- I don't get its importance in this article, even if it were to be a link, it won't work! Just do away with all of them in this article!

**Review by jumamiller — CHANGES_REQUESTED** [review]

File: articles/understanding-static-dynamic-routing/index.md (hunk 5 above)
Comment (raboba2re) [reply articles/understanding-static-dynamic-routing/index.md]: OK Sir

**Comment by jumamiller** [comment]
@raboba2re, review is done, add your author details( in case you've not).

**Comment by jumamiller** [comment]
> @raboba2re, review is done, add your author details( in case you've not).

> I realized this could be your "first article" since the other one is not yet ready for final review!

**Comment by jumamiller** [comment]
> > @raboba2re, have a look at the image sources, they aren't working! Better still, reference them at the end of the article!
> 
> @jumamiller The author owns the images and I have made changes to reflect that.

> Authors are advised to use non copyrighted images in their article! Just an FYI!!!!

**Comment by jumamiller** [comment]
ops! Author details are added separately in the author's directory! read more [here](https://github.com/section-engineering-education/engineering-education/tree/master/new_contributors).

**Comment by raboba2re** [comment]
> 
> 
> ops! Author details are added separately in the author's directory! read more [here](https://github.com/section-engineering-education/engineering-education/tree/master/new_contributors).

@jumamiller  the author's directory has been created from my first submission. Thank you .

**Comment by jumamiller** [comment]
> > ops! Author details are added separately in the author's directory! read more [here](https://github.com/section-engineering-education/engineering-education/tree/master/new_contributors).
> 
> @jumamiller the author's directory has been created from my first submission. Thank you .

> oh thanks for clarification!

**Comment by jumamiller** [comment]
@hectorkambow, this article is ready for final review!

**Comment by raboba2re** [comment]
@jumamiller , @zolomohan I have made some serious changes, can you please go through it?

**Comment by hectorkambow** [comment]
[Understanding static and dynamic routing #1520.pdf](https://github.com/section-engineering-education/engineering-education/files/6038354/Understanding.static.and.dynamic.routing.1520.pdf)


👋  Good afternoon @raboba2re  

Upon running this article through our 3rd party plagiarism checker it raised a few red flags and the % was higher than we typically like to accept. 

Please revisit the article to make sure we are contributing unique and original content to the community and that we are adding any sources (if we haven't already). 

cc @jumamiller  just an FYI for potential future revisions - please make sure to add the plagiarism label to the PR before doing the review to help save time and effort 👍

**Comment by hectorkambow** [comment]
@raboba2re  Also want to make sure that the images do not need any sources.

**Comment by raboba2re** [comment]
> 
> 
> @raboba2re Also want to make sure that the images do not need any sources
@hectorkambow  Only two images require source, and the sources have been added.

**Comment by raboba2re** [comment]
@hectorkambow The article has been reviewed and updated to address your concerns.

**Comment by jumamiller** [comment]
@raboba2re, I noticed you constantly make changes! When an article has been marked "ready for final review" do not make changes unless you're asked to do so! an FYI

**Comment by raboba2re** [comment]
> 
> 
> @raboba2re, I noticed you constantly make changes! When an article has been marked "ready for final review" do not make changes unless you're asked to do so! an FYI

Good day @jumamiller , @hectorkambow  requested  some changes to address  plagiarism issues.

**Comment by raboba2re** [comment]
> 
> 
> [Understanding static and dynamic routing #1520.pdf](https://github.com/section-engineering-education/engineering-education/files/6038354/Understanding.static.and.dynamic.routing.1520.pdf)
> 
> wave Good afternoon @raboba2re
> 
> Upon running this article through our 3rd party plagiarism checker it raised a few red flags and the % was higher than we typically like to accept.
> 
> Please revisit the article to make sure we are contributing unique and original content to the community and that we are adding any sources (if we haven't already).
> 
> cc @jumamiller just an FYI for potential future revisions - please make sure to add the plagiarism label to the PR before doing the review to help save time and effort +1

@jumamiller

**Comment by jumamiller** [comment]
@hectorkambow, I'm not sure whether this article is ready for final review due to the changes made(by the contributor) without my knowledge. I will have to review this article again!
@raboba2re, when you're done with plagiarism issues, DO NOT make changes unless I ask you to do so!

**Comment by jumamiller** [comment]
@raboba2re, the article looks good, my only concern is images. Use NON-COPYRIGHTED images in your article without sources indicated.  
You can get free images [here](https://unsplash.com/) for use in your article.

**Comment by raboba2re** [comment]
@jumamiller , Thank you for your time. 
@jumamiller  @hectorkambow I own the copyright to all the images except two, which I referenced in the article.

**Comment by jumamiller** [comment]
@hectorkambow, he has worked on plagiarism, you can proceed with the final review (**pay attention to images used**).Thanks!

**Comment by raboba2re** [comment]
@jumamiller I have changed the image

**Comment by hectorkambow** [comment]
@raboba2re  

Due to the increased level of revisions - this article qualifies for a tiered payout. 

Please reference our [FAQ ](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/FAQ.md)for more info on tiered payouts 👍  

Thank you

**Pull request merged**